### PR TITLE
Fixes #21721: Missing ubuntu 22.04 support in documentation

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/21721

### CODE REVIEW

File: src/reference/modules/installation/pages/operating_systems.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ available freely for these platforms:
 | OS | Version | Architecture
 
 | Debian | 10 and 11 | 64bit
-| Ubuntu | 18.04 LTS and 20.04 LTS | 64bit
```
Comment (VinceMacBuche) [inline src/reference/modules/installation/pages/operating_systems.adoc:21]: you need to keep ubuntu 20.04

**Review by VinceMacBuche — CHANGES_REQUESTED** [review]

File: src/reference/modules/installation/pages/operating_systems.adoc (hunk 1 above)
Comment (ElaadF) [reply src/reference/modules/installation/pages/operating_systems.adoc:21]: 18.04 to 22.04 is ok ?

Comment (VinceMacBuche) [reply src/reference/modules/installation/pages/operating_systems.adoc:21]: no it may induced that we suport non LTS version

**Comment by ElaadF** [comment]
Commit modified

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**